tests/datafile/parkes.toa

```diff
@@ -1,9 +1,9 @@
 # These are Parkes format and have been changed to also be Parkes observatory
```

Comment:
I added that comment about changing the observatory to Parkes.  The comment needs updated as it is no longer correct!